xcodeproj/internal/files/incremental_resources.bzl

```diff
@@ -9,6 +9,14 @@ load(":files.bzl", "join_paths_ignoring_empty")
 
 # Utility
 
+_FOLDER_TYPE_FILE_SUFFIXES = [
+    ".bundle/",
+    ".docc/",
+    ".framework/",
+    ".scnassets/",
+    ".xcassets",
```

Comment:
Mind adding a comment about why `/` isn't present in this one vs. the rest?

Reply:
Bug